src/Math-Matrix/PMNDArray.class.st

```diff
@@ -41,7 +41,7 @@ ifTrue:[
 { #category : #comparing }
 PMNDArray >> = anArray [
 
```

Comment:
Maybe use ``aPMNDArray`` instead of ``anArray``, because this is not an instance of ``Array`` class.

Reply:
Ok. Thanks